base/src/commonMain/kotlin/monitor/ServiceMonitor.kt

```diff
@@ -23,7 +23,7 @@ import kotlinx.coroutines.flow.asStateFlow
 
 abstract class ServiceMonitor {
 
-    abstract val TAG: String
+    protected val TAG: String = this::class.simpleName!!
```

Comment:
:+1:

Reply:
Worth noting that at least in older Kotlin/Native version (talking almost 2 years ago here) if you would extend a class from the iOS side `simpleName` _would_ be null. Always safer to do ? "ServiceMonitor" rather than a forced unwrap.

Reply:
dea80fc